test/controllers/teams_controller_test.rb

```diff
@@ -305,6 +305,11 @@ def setup
     assert_select 'h3', 'Per User Statistics'
   end
 
+  test 'create team with location' do
```

Comment:
This is not a controller test since it doesn't touch anything in the controller, this only touches things in the model. A proper controller test would actually change the data by hitting the controller

https://github.com/mitre-cyber-academy/ctf-scoreboard/blob/34bcf44c2dff0d73cf5f4b3a913c4e5f546c4626/test/controllers/teams_controller_test.rb#L190-L196

and then verify afterwards that 

```ruby
assert 'new_location', team.reloads.team_location
```

You can modify the exact test you have above to add these checks.